webview-ui/src/components/chat/ChatTextArea.tsx

```diff
@@ -790,10 +796,17 @@ const ChatTextArea = forwardRef<HTMLTextAreaElement, ChatTextAreaProps>(
 										{config.name}
 									</option>
 								))}
-								<option disabled style={{ borderTop: "1px solid var(--vscode-dropdown-border)" }}>
+								<option
```

Comment:
Nice fix! There's one more case above this where you can replace the style block with optionStyle - mind doing that and then I'll merge?